source/common/http/http1/codec_impl.cc

```diff
@@ -231,7 +226,7 @@ void StreamEncoderImpl::encodeHeadersBase(const RequestOrResponseHeaderMap& head
     }
   }
 
-  connection_.addToBuffer(CRLF);
+  connection_.buffer().addFragments(CRLF);
```

Comment:
For single argument cases we're better off calling add

Reply:
It's ok. I have no particular insistence on this.